src/main/java/godaddy/registry/jtoolkit2/se/fee/DomainCreateFeeCommandExtension.java

```diff
@@ -42,6 +43,11 @@ public DomainCreateFeeCommandExtension(BigDecimal registrationFee, String curren
 
     }
 
+    public DomainCreateFeeCommandExtension(BigDecimal registrationFee, String currency, boolean descriptionRequired) {
+        this(registrationFee, currency);
+        this.descriptionRequired = descriptionRequired;
+    }
```

Comment:
Richard pointed out that - it would be better to swap the body so that new Constructor would be doing all the work.

And for the old c-tor, it would be simply changed to:
```
    public DomainCreateFeeCommandExtension(BigDecimal registrationFee, String currency) {
        this(registrationFee, currency, true);
    }
```

So that we get a few benefits:
1. Do not have to initialise the boolean field to `true` 
2. Also can make that field as `final`.

Reply:
Done